code/game/area/areas.dm

```diff
@@ -390,7 +390,7 @@ var/list/mob/living/forced_ambiance_list = new
 	else if(A && A.has_gravity)
 		return 1
 	else
-		if(T && SSmachinery.gravity_generators["[T.z]"] && length(SSmachinery.gravity_generators["[T.z]"]))

```

Comment:
This can be simplified to `T && length(SSmachinery.gravity_generators)`, `length(null) == 0`.